README.md

```diff
@@ -45,32 +47,75 @@ The overall goal is to create an easily self-hostable, decentralized alternative
 Each lemmy server can set its own moderation policy; appointing site-wide admins, and community moderators to keep out the trolls, and foster a healthy, non-toxic environment where all can feel comfortable contributing.
 
 ## Why's it called Lemmy?
+
 - Lead singer from [motorhead](https://invidio.us/watch?v=pWB5JZRGl0U).
-- The old school [video game](https://en.wikipedia.org/wiki/Lemmings_(video_game)).
+- The old school [video game](<https://en.wikipedia.org/wiki/Lemmings_(video_game)>).
 - The [furry rodents](http://sunchild.fpwc.org/lemming-the-little-giant-of-the-north/).
 
 Made with [Rust](https://www.rust-lang.org), [Actix](https://actix.rs/), [Inferno](https://www.infernojs.org), [Typescript](https://www.typescriptlang.org/) and [Diesel](http://diesel.rs/).
 
-## Install
-### Docker
+## Usage
+
+### Production
+
+#### Docker
+
 Make sure you have both docker and docker-compose installed.
+
 ```
 git clone https://github.com/dessalines/lemmy
 cd lemmy
 ./docker_update.sh # This pulls the newest version, builds and runs it
 ```
+
 and goto http://localhost:8536
-### Local Development
-#### Requirements
+
+<!-- #### Kubernetes (WIP)
```

Comment:
We can just wait to merge all this until you're ready for it.

Reply:
That's probably a good idea, a production version should exist beforehand.